app/helpers/me/notices_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Me::NoticesHelper
+  def dissmiss_all_notices_button
+    link_to_remote_with_confirm(:dismiss_all_notices.t,
```

Comment:
hmm then we should make it so the confirm option of link_to also uses the nice modal version.